test/IntelFPGAMemoryAttributes.ll

```diff
@@ -150,6 +150,25 @@
 ;   s.field = 0;
 ; }
 ;
+; void force_pow2_depth_attr() {
+;   [[intelfpga::force_pow2_depth(0)]] int fp2d_var;
+;
+;   [[intelfpga::force_pow2_depth(1)]] struct fp2d_st {
```

Comment:
Could we have a test-case for `force_pow2_depth(0)` on a `struct`?

Reply:
We may have it, but for what reason? From translator's point of view processing of attribute with any argument value is not depend on a type of variable.

Reply:
Fair enough - just my morning safe-guard mood speaking :)